dists/scummvm.rc

```diff
@@ -106,7 +107,7 @@ BEGIN
             VALUE "FileDescription", "ScummVM: A free implementation of various adventure game engines\0"
             VALUE "FileVersion", SCUMMVM_VERSION "\0"
             VALUE "InternalName", "scummvm\0"
-            VALUE "LegalCopyright", "Copyright � 2001-2019 The ScummVM Team\0"
+            VALUE "LegalCopyright", "Copyright � 2001-2019 The ScummVM Team\0"
```

Comment:
Why is this line changed? Was the encoding of the file changed maybe?
The `?` character here is a copyright character, and the encoding of the file is I think ISO 8859-1 originally.

Reply:
This is most likely cause by Visual Studio Code which for whatever reason decided to treat this file as UTF-8. I'll revert it to ISO 8859-1.

Reply:
Fixed by 6b0dd0c.